openfaas/post_install.md

```diff
@@ -27,7 +45,13 @@ export KUBECONFIG=$HOME/Downloads/config-file.yaml
 
 ### Find your generated password
 
-Now retrieve your password from above and save it into a file called `password.txt`.
+You can find your password in the Civo Dashboard by clicking on the OpenFaaS application and save it as `password.txt`.
```

Comment:
You don't need to say "You can find your password in the Civo Dashboard by clicking on the OpenFaaS application" because this is inherently only displayed in the Civo Dashboard, when they have already clicked the installed OpenFaaS application. That's why I changed it to "from above".

Reply:
No problem. I thought you may use this text in other places, like the CLI.